src/main/java/com/kodilla/ecommercee/domain/Cart.java

```diff
@@ -28,5 +28,5 @@
             joinColumns = {@JoinColumn(name = "CART_ID", referencedColumnName = "CART_ID")},
```

Comment:
nie wiem czy dobrze zrozumiałam to co pisałeś w moim pull requeście ale spodziewałam się że tutaj będziesz miał mappedBy?

Reply:
no właśnie ja zrobiłem odwrotnie, @JoinTable w Cart, a "mappedBy" w Product